src/op_mode/restart_dhcp_relay.py

```diff
@@ -47,6 +51,9 @@
         if not c.exists_effective('service dhcpv6-relay'):
             print("DHCPv6 relay service not configured")
         else:
+            if commit_in_progress():
+                print('Cannot restart DHCP-relay as commit is still in progress')
```

Comment:
`Can not restart DHCPv6 relay until commit in progress as a commit is still in progress`...

Reply:
"Cannot restart DHCPv6 relay while a commit is in progress"/

Reply:
Done!